content/md/en/docs/learn/consensus.md

```diff
@@ -116,7 +116,7 @@ The difficulty of this problem can be tuned to provide a statistical target bloc
 
 ### GRANDPA
 
-[GRANDPA](https://paritytech.github.io/substrate/master/sc_finality_grandpa/index.html) provides block finalization.
+[GRANDPA](https://crates.io/crates/finality-grandpa) provides block finalization.
```

Comment:
I think the exact same page like the last one is this:
https://paritytech.github.io/substrate/master/sc_consensus_grandpa/index.html

Reply:
Pushed already 🤦🏽 sorry about that - let me know if you want me to revert to `https://crates.io/crates/finality-grandpa`

Reply:
I'll replace since the one above gives:
```
 [✖] https://crates.io/crates/finality-grandpa → Status: 404
 ```
 even though it exists